pkgs/applications/networking/irc/weechat/default.nix

```diff
@@ -28,25 +28,16 @@ let
   in
     assert lib.all (p: p.enabled -> ! (builtins.elem null p.buildInputs)) plugins;
     stdenv.mkDerivation rec {
-      version = "3.2.1";
+      version = "3.3";
       pname = "weechat";
 
       hardeningEnable = [ "pie" ];
 
       src = fetchurl {
         url = "https://weechat.org/files/src/weechat-${version}.tar.bz2";
-        sha256 = "sha256-gtFskJoJLw7cqLao46LRUcZobOtzgwUOiKf7I4OiJAQ=";
+        sha256 = "sha256-GnSi7uMxiyWSQau75q07NlX1ikaBeWOdrzOf9f0jnBM=";
       };
 
-      patches = [
-        # weechat 3.2 fails to build on Darwin, but is fixed for the next release:
-        (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Pushed.